components/servo/examples/winit_minimal.rs

```diff
@@ -37,6 +35,10 @@ fn main() -> Result<(), Box<dyn Error>> {
     let mut app = App::new(&event_loop);
     event_loop.run_app(&mut app)?;
 
+    if let App::Running { servo, .. } = app {
+        servo.deinit();
```

Comment:
I hope eventually we can use drop trait to handle all these, including webrender's deinit.

Reply:
We were actually exploring using the `Drop` trait to do this and had a version, but we had a lot of doubts as `Servo::deinit()` can block on thread joining. We weren't sure if it made sense to have that kind of blocking behavior in a drop impl, so we decided to wait to think and discuss with everyone a bit more.